src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -95,20 +113,23 @@ static void LaunchAndWait(Permutation permutation, TestDescriptor testDescriptor
             ProcessStartInfo pi;
             if (permutation.Platform == Platform.NetFramework)
             {
+                // ReSharper disable once RedundantAssignment
+                var processIdArgument = string.Empty;
+#if NET452
                 var processId = DebugAttacher.GetCurrentVisualStudioProcessId();
-                var processIdArgument = processId >= 0 ? $" --processId={processId}" : string.Empty;
-
+                processIdArgument = processId >= 0 ? $" --processId={processId}" : string.Empty;
+#endif
                 pi = new ProcessStartInfo(testDescriptor.ProjectAssemblyPath, permutationArgs + sessionIdArgument + processIdArgument)
                 {
-                    UseShellExecute = false,
+                    UseShellExecute = true,
```

Comment:
This is probably going to cause issues as now the test will be launched in a new console.

Reply:
I had to enable this when running the test (not the endpoint) on .net core, as there was no console showing up at all. This might be something we have to #ifdef as well.

Reply:
testing this a bit more explicitly, it doesn't seem to cause any issues or spawn new consoles when running this with UseShellExecute = true.